src/test/java/no/unit/nva/model/contexttypes/ReportTest.java

```diff
@@ -35,15 +35,15 @@
     @DisplayName("Reports can be created")
     @ParameterizedTest
     @CsvSource({
-        "Series,123,A publisher,9780201309515,0363-6941,1945-662X",
-        "Series,123,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
-        "Series,123,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
-        "Series,123,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
-        "Series,,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
+        "Journal,123,A publisher,9780201309515,0363-6941,1945-662X",
```

Comment:
I recommend changing the value to "Journal title" and changing the name of the respective parameter as well

Reply:
Fixed in following PR